lib/utils/index.ts

```diff
@@ -66,6 +66,11 @@ const ASYNC_UTILS = [
   'waitForDomChange',
 ] as const;
 
+const DEBUG_UTILS = [
+  'debug', //
```

Comment:
There is an empty comment at the end of the line, I guess is a leftover 😅

Reply:
No actually this is a marker for `prettier` to have it put the array items on their own lines even if it can fit on a single line with 80 chars.

I find that useful for lists like these where it's known that more items will be added later, since it reduces the diff.

Happy to remove this if  you want it formatted to a single line, or otherwise happy to add a few more utils (which was my original plan).

Reply:
Oh wow, I didn't know this way of forcing prettier to format array elements per line! It's still weird tho, but since we are gonna add more utils, I guess it will disappear.